### DISCUSSION TITLE
Allow Filtering by Domain

### DISCUSSION THREAD

**Discussion by justaguydoinghisstuff** [simple-login/app#450] [Feature Requests]
Hello,
I would like to suggest the option of filtering the alisases by domain. It would come as helpful when managing aliases or you simply want to have a overview of your aliases.
I.e If one has 2 domains and simple login defaut domains. then he could choose to show/hide aliases from a particular domain.

**Comment by hiccups1980** [comment]
That would be great. I would love to filter by domain and subdomain.
